features/support/helpers.rb

```diff
@@ -149,13 +154,13 @@ def seconds_agnostic_datetime(datetime = Time.now)
     "#{time}:\\d{2}",
     Regexp.escape(zone)
   ] \
-  .join("\\ ")
+    .join("\\ ")
 end
 
 #
 
 def seconds_agnostic_time(time)
   time = time.strftime("%H:%M:%S") if time.is_a?(Time)
-  hour, minutes, _ = time.split(":")
+  hour, minutes, = time.split(":")
```

Comment:
The other way seemed clearer to me too, but Rubocop wants what Rubocop wants.

Maybe when things settle down, I will dig into this a bit more to figure out _why_ RC wanted this changes.